### PR TITLE
Update testing docs to reflect --grep syntax

### PR DESCRIPTION
**Pull request by nornagon**
Looks like the -match syntax is no longer supported.

### CODE REVIEW

File: docs/development/testing.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ app (surprise!) that can be found in the `spec` folder. Note that it has
 its own `package.json` and that its dependencies are therefore not defined
 in the top-level `package.json`.
 
-To run only a selected number of tests, run `npm run test -match=NAME`,
-replacing the `NAME` with the file name of the test suite you would like
-to run. As an example: If you want to run only IPC suites, you would run
-`npm run test -match=ipc`.
+To run only specific tests matching a pattern, run `npm run test --
+-g=PATTERN`, replacing the `PATTERN` with a regex that matches the test suites
```
Comment (zeke) [inline docs/development/testing.md]: test suites -> tests.

The "suite" generally refers to all the tests. Otherwise, looks good to me 👍

**Comment by zeke** [comment]
👌

**Comment by codebytere** [comment]
do we want to trop this

**Comment by zeke** [comment]
> do we want to trop this

I would say no, since we're already surfacing electron/electron#master tutorial content on the website: https://electronjs.org/docs/development/testing

**Pull request merged**